BaseFilter/sections/specific.txt

```diff
@@ -566,6 +566,13 @@ $subdocument,third-party,denyallow=disqus.com,domain=welovemanga.one
 ! NOTE: Specific rules
 !
 !
+ww2-weapons.com###sgpb-popup-dialog-main-div-wrapper
+ww2-weapons.com##.sgpb-theme-6-overlay
+ww2-weapons.com##.widget.warhist-widget
```

Comment:
```suggestion
ww2-weapons.com##.warhist-widget
```
`.warhist-widget` is enough. Or `div[id^="warhist-"]` - it will cover also `div[class="warhist-after-content"]` (it will be unnecessary)